src/metadata/matroska_handler.cc

```diff
@@ -256,11 +256,11 @@ std::string MatroskaHandler::getContentTypeFromByteVector(const LIBMATROSKA_NAME
 #endif
 }
 
-void MatroskaHandler::addArtworkResource(const std::shared_ptr<CdsItem>& item, const std::string& art_mimetype)
+void MatroskaHandler::addArtworkResource(const std::shared_ptr<CdsItem>& item, std::string_view art_mimetype)
 {
```

Comment:
This change produces an extra object because its used in call to renderProtocolInfo as string&

Reply:
Removed.